README.md

```diff
@@ -115,7 +122,7 @@ The same steps should be followed for receiver and subnet tests as we are showin
    - To run the simulation, type in the terminal
```

Comment:
No information/steps provided on how to check the output of task 4. Pls add.

Reply:
Changes to be done in installation guide: 
After modularizing the components as submodule, steps are not updated on how to clone the project along with submodules. Pls add the steps

Reply:
Done